parts/dcoscustomdata190.t

```diff
@@ -303,3 +304,42 @@ write_files:
   content: 'ATTRIBUTES_STR'
   permissions: "0644"
   owner: "root"
+- content: |
+    #!/bin/bash
+    # Check the partitions on /dev/sdb created by cloudinit and force a detach and
+    # reformat of the parition.  After which, all will be remounted.
+    EPHEMERAL_DISK="/dev/sdb"
+    PARTITIONS=`fdisk -l $EPHEMERAL_DISK | grep "^$EPHEMERAL_DISK" | cut -d" " -f1 | sed "s~$EPHEMERAL_DISK~~"`
+    if [ -n "$PARTITIONS" ]; then
+        for f in $PARTITIONS; do
+            df -k | grep "/dev/sdb$f"
+            if [ $? -eq 0 ]; then
+                umount -f /dev/sdb$f
+            fi
+            mkfs.ext4 /dev/sdb$f

```

Comment:
what latency does this add to the deployment?  For example, we lazy format our attached disks: https://github.com/Azure/acs-engine/blob/master/parts/kubernetesmastercustomdata.yml#L294.